Gemfile

```diff
@@ -16,7 +16,8 @@ gem 'keen'
 
 gem 'pushpop'
 gem 'pushpop-keen'
-gem 'pushpop-sendgrid'
+gem 'pushpop-sendgrid', :git => 'git@github.com:pushpop-project/pushpop-sendgrid.git'
```

Comment:
This is causing Travis CI issues. We can try and get permissions set for this gem to update it today.